### ISSUE TITLE
MIT License

### ISSUE THREAD

**Issue by by14** [xitanggg/open-resume#45]
I'm hoping to fork this and make basic modifications for some of my friends to use [a different template, different fonts, etc.]. Is it possible to make it an MIT license instead? Thanks!

**Comment by p0lygun** [comment]
AGPL allows you to modify the code as you see fit, but you will need to keep it open-source

**Comment by by14** [comment]
Right, I'm hoping to integrate it with one of my other small projects [a job watcher] which I'm hoping not to have to open source at this time.
